network/rtl8188fu/rtl8188fu.SlackBuild

```diff
@@ -0,0 +1,80 @@
+#!/bin/bash
+
+# Slackware build script for rtl8188fu
+# Written by Juan M. Lasca <juanmlasca@gmail.com>
+# Licensed under the WTFPL. See http://www.wtfpl.net/txt/copying/ for details.
+# Based on rtl8812bu.SlackBuild
+
+cd $(dirname $0) ; CWD=$(pwd)
+
+PRGNAM=rtl8188fu
+VERSION=${VERSION:-1.0+20231018_68ced40}
+COMMIT=68ced40d862d13663294496bac2e9a91ffa0e5c7
+BUILD=${BUILD:-1}
+TAG=${TAG:-_SBo}
+PKGTYPE=${PKGTYPE:-tgz}
+
+KERNEL=${KERNEL:-$(uname -r)}
+PKGVER=${VERSION}_$(echo $KERNEL | tr - _)
+
+if [ -z "$ARCH" ]; then
+    case "$( uname -m )" in
+        i?86) ARCH=i586 ;;
+        arm*) ARCH=arm ;;
+        *) ARCH=$( uname -m ) ;;
+    esac
+fi
+
+if [ ! -z "${PRINT_PACKAGE_NAME}" ]; then
+    echo "$PRGNAM-$PKGVER-$ARCH-$BUILD$TAG.$PKGTYPE"
+    exit 0
+fi
+
+TMP=${TMP:-/tmp/SBo}
+PKG=$TMP/package-$PRGNAM
+OUTPUT=${OUTPUT:-/tmp}
+
+set -e
+
+rm -rf $PKG
+mkdir -p $TMP $PKG $OUTPUT
+cd $TMP
+rm -rf $PRGNAM-$COMMIT
+tar xvf $CWD/$PRGNAM-$COMMIT.tar.gz
+cd $PRGNAM-$COMMIT
+chown -R root:root .
+find -L .  -perm /111 -a \! -perm 755 -a -exec chmod 755 {} \+ -o \
+     \! -perm /111 -a \! -perm 644 -a -exec chmod 644 {} \+
+
+make
+
+install -D -m0644 rtl8188fu.ko \
+        $PKG/lib/modules/$KERNEL/kernel/drivers/net/wireless/rtl8188fu.ko
+
+mkdir -p $PKG/lib/modprobe.d/
+mkdir -p $PKG/etc/NetworkManager/conf.d/
+
+# disable power management
```

Comment:
These should all use `install` with correct user/group + permissions instead of `cp`